suite2p/io/nwb.py

```diff
@@ -262,153 +262,164 @@ def read_nwb(fpath):
     return stat, ops, F, Fneu, spks, iscell, probcell, redcell, probredcell
 
 
-def save_nwb(save_folder):
+def create_nwb_file(ops, multiplane):
+    ### INITIALIZE NWB FILE
+    nwbfile = NWBFile(
+        session_description='suite2p_proc',
+        identifier=str(ops['data_path'][0]),
+        session_start_time=(ops['date_proc'] if 'date_proc' in ops 
+                            else datetime.datetime.now())
+    )
+    print(nwbfile)
+
+    device = nwbfile.create_device(
+        name='Microscope', 
+        description='My two-photon microscope',
+        manufacturer='The best microscope manufacturer'
+    )
+    optical_channel = OpticalChannel(
+        name='OpticalChannel', 
+        description='an optical channel', 
+        emission_lambda=500.
+    )
+
+    imaging_plane = nwbfile.create_imaging_plane(
+        name='ImagingPlane',
+        optical_channel=optical_channel,
+        imaging_rate=ops['fs'],
+        description='standard',
+        device=device,
+        excitation_lambda=600.,
+        indicator='GCaMP',
+        location='V1',
+        grid_spacing=([2.0,2.0,30.0] if multiplane else [2.0,2.0]),
+        grid_spacing_unit='microns'
+    )
+
+    # link to external data
+    image_series = TwoPhotonSeries(
+        name='TwoPhotonSeries', 
+        dimension=[ops['Ly'], ops['Lx']],
+        external_file=(ops['filelist'] if 'filelist' in ops else ['']), 
+        imaging_plane=imaging_plane,
+        starting_frame=[0], 
+        format='external', 
+        starting_time=0.0, 
+        rate=ops['fs'] * ops['nplanes']
+    )
```

Comment:
Could we include a way of specifying this metadata?

Reply:
Likely, though it's outside the scope here.  I am not trying to change any existing behavior -- just trying to add functionality.